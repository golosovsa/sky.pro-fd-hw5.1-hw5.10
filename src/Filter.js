import ButtonText from "./ButtonText"

function Filter() {
  return (
    <>
      <div className="filter__title">Искать по:</div>
      <ButtonText text="исполнителю" classes="filter__button button-author"/>
      <ButtonText text="году выпуска" classes="filter__button button-year"/>
      <ButtonText text="жанру" classes="filter__button button-genre"/>
    </>
  )
}

export default Filter
