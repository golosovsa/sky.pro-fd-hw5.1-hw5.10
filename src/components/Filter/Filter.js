import FilterContainer from "../styled/FilterContainer/FilterContainer";
import Button from "../styled/Button/Button";

function Filter() {
    return (
        <FilterContainer>
            Искать по:
            <FilterContainer.Buttons>
                <Button>исполнителю</Button>
                <Button>году выпуска</Button>
                <Button>жанру</Button>
            </FilterContainer.Buttons>
        </FilterContainer>
    );
}

export default Filter;
