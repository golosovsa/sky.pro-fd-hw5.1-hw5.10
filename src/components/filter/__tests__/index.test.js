import ShallowRenderer from 'react-test-renderer/shallow'
import Filter from '..'

describe('<Filter />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<Filter>123</Filter>)

      expect(tree).toMatchSnapshot()
    })
  })
})
