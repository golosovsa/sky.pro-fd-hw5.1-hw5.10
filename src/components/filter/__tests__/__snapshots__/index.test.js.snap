// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filter /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div>
  <styled.p>
    Искать по:
  </styled.p>
  <styled.div>
    123
  </styled.div>
</styled.div>
`;
